po/pl.po

```diff
@@ -671,6 +695,9 @@ msgstr "Kanal:"
 msgid "Chaumont"
 msgstr "Chaumont"
 
+msgid "Check on startup"
+msgstr "Sprawdzać przy starcie"
```

Comment:
As with the previous comment, "sprawdzanie" or "sprawdzaj" could sound a little bit more natural. It feels a bit strange (at least for me) when infinitive forms (such as "sprawdzać") are used in user interface and, for some reason, they seem to be very frequently used by various malicious/scamming websites which may be one of the reasons why they sound wrong in this context.

Reply:
```suggestion
msgstr "Sprawdzanie przy starcie"
```